serving_local_nlp_model.py

```diff
@@ -175,6 +175,10 @@ def _run_inference(self):
             else:
                 logprobs_buffer = None
 
+            # fixes error on Falcon models
+            if self.return_token_type_ids is False:
+                self.tokenizer.return_token_type_ids = False
+
```

Comment:
Two more things:

```suggestion
            # fixes error on Falcon models
            if not self.return_token_type_ids:
                self.tokenizer.return_token_type_ids = False

```

And why not set this in the `__init__` constructor instead of here?